src/app/containers/NielsenAnalytics/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import { RequestContext } from '#contexts/RequestContext';
+import useToggle from '#hooks/useToggle';
+import AmpNielsenAnalytics from './Amp';
+
+const NielsenAnalytics = () => {
+  const { service, isAmp, pathname } = useContext(RequestContext);
+
+  // nielsenAnalytics toggle only set up as local toggle at the moment...
+  const { enabled } = useToggle('nielsenAnalytics');
+
+  if (!enabled || !isAmp) {
+    return null;
+  }
+
+  // News and Sport only
+  let apid;
+  if (service === 'news') {
+    apid = '474C2B0B-0C04-4182-BCFB-FC9469A48C9B';
+  }
+  if (service === 'sport') {
+    apid = 'DC4AFDB2-B352-4D51-81EF-38BE41114F22';
+  }
+  if (!apid) {
+    return null;
+  }
+
+  // strip .amp off the path
+  // i.e. CPS /news/business-58007120
+  // i.e. Optimo /news/articles/c6v11qzyv8po
+  const assetid = pathname.split('.')[0];
+
+  const capitalise = word => {
+    return `${word[0].toUpperCase()}${word.substr(1)}`;
+  };
+
+  // CPS ["", "news", "business-58007120"]
+  // Optimo ["", "news", "articles", "c6v11qzyv8po"]
+  let section = `BBC ${capitalise(service)}`;
+  const sectionSplit = assetid.split('/');
+  const lastSection = sectionSplit[sectionSplit.length - 1];
+  if (lastSection.includes('-')) {
+    // CPS
+    const lastSectionSplit = lastSection.split('-');
+    section = `${section} ${capitalise(lastSectionSplit[0])}`;
+  }
+
+  return (
+    <AmpNielsenAnalytics apid={apid} assetid={assetid} section={section} />
+  );
+};
```

Comment:
I think the logic in here should be explicitly tested missing coverage includes:
- Correct `apid` for news and sport services
- Removing `.amp` from the `assetid`
- Logic to determine the `section`, I can't really understand fully what it's intent is

Reply:
I've added tests for the apid.
I'd rather leave the assetid ones for now - there are ongoing conversations with Nielsen on what these should be and why the currently aren't getting sent.

Reply:
Okay @TimHewitt , could you please revisit in a later PR to add them? Given this is an area we as a team have little knowledge of having tests to describe the requirements is even more important.